docs/pages/desktop-access/getting-started.mdx

```diff
@@ -1,214 +1,171 @@
+<TabItem label="System Package">
 
 ```code
-$ systemctl start teleport.service
+$ sudo systemctl start teleport.service
 ```
 
-Copy the join token to a file on the instance where you will run the Windows Desktop
-Service, and then use the following configuration:
+</TabItem>
+<TabItem label="Tar Archive">
 
-```yaml
-version: v3
-teleport:
-  auth_token: /path/to/token
-  proxy_server: teleport.example.com # replace with your proxy address
-windows_desktop_service:
-  enabled: yes
-  ldap:
-    # Port must be included for the addr.
-    # LDAPS port is 636 by default,
-    # e.g. example.com:636
-    addr: "$LDAP_SERVER_ADDRESS"
-    domain: "$LDAP_DOMAIN_NAME"
-    username: '$LDAP_USERNAME'
-    # This should be the path to the certificate exported in Step 4.
-    der_ca_file: /path/to/cert
-  discovery:
-    base_dn: "*"
-auth_service:
-  enabled: no
-proxy_service:
-  enabled: no
-ssh_service:
-  enabled: no    
+```code
+$ sudo teleport install systemd --output=/etc/systemd/system/teleport.service;
+$ sudo systemctl enable teleport;
+$ sudo systemctl start teleport;
 ```
+
 </TabItem>
-<TabItem label="Standalone Binary">
+</Tabs>
 
-From the directory containing the Teleport binary:
+## Step 3/4. Configure Windows access
 
-```code
-$ ./teleport start --config=/etc/teleport.yaml
-```
+In order to gain access to a remote desktop, a Teleport user needs to have the
+appropriate permissions for that desktop.
 
-Copy the join token to a file on the instance where you will run Windows Desktop
-Service, and then use the following configuration:
+Create the file `windows-desktop-admins.yaml`:
 
 ```yaml
-version: v3
-teleport:
-  auth_token: /path/to/token
-  proxy_server: mytenant.teleport.sh # replace with your cloud tenant
-windows_desktop_service:
-  enabled: yes
-  ldap:
-    # Port must be included for the addr.
-    # LDAPS port is 636 by default,
-    # e.g. example.com:636
-    addr: "$LDAP_SERVER_ADDRESS"
-    domain: "$LDAP_DOMAIN_NAME"
-    username: '$LDAP_USERNAME'
-    # This should be the path to the certificate exported in Step 5.
-    der_ca_file: /path/to/cert
-  discovery:
-    base_dn: "*"
-auth_service:
-  enabled: no
-proxy_service:
-  enabled: no
-ssh_service:
-  enabled: no    
+kind: role
+version: v6
+metadata:
+  name: windows-desktop-admins
+spec:
+  allow:
+    windows_desktop_labels:
+      "*": "*"
+    windows_desktop_logins: ["Administrator", "alice"]
 ```
-</TabItem>
-</Tabs>
 
-The access wizard will detect when the new Teleport instance has joined the
-cluster, and you can then click **Next**.
+You can restrict access to specific hosts by defining values for
+`windows_desktop_labels`, and adjust the array of usernames this role has access
+to in `windows_desktop_logins`.
+
+<Admonition type="warning" title="RBAC Configuration">
+Ensure that each Teleport user is only assigned Windows logins that they should
+be allowed to access.
+
+Usernames shared between domain and local users will create login conflicts.
```

Comment:
Since this guide is about non-AD, I don't think domain users are are play. Should we remove this?